auth-api/src/auth_api/services/affiliation.py

```diff
@@ -293,9 +305,14 @@ def get_nr_payment_details(business_identifier):
         return invoices
 
     @staticmethod
-    def delete_affiliation(org_id, business_identifier, email_addresses: str = None,
-                           reset_passcode: bool = False, log_delete_draft: bool = False):
+    def delete_affiliation(delete_affiliation_request: DeleteAffiliationRequest, environment: str = None):
```

Comment:
suggestion: `delete_affiliation_request` -> `request`, that way you don't have to remap like on 310